recipe/meta.yaml

```diff
@@ -35,6 +36,7 @@ requirements:
     - python
     - {{ pin_compatible('numpy') }}
     - {{ pin_compatible('scipy') }}
+    - scipy
```

Comment:
Is there a reason why we need both the {{ pin_compatible('scipy') }} and scipy?  Just curious here.

Reply:
there wasn't, addressed